freqtrade/freqtradebot.py

```diff
@@ -1117,7 +1118,7 @@ def _notify_sell_cancel(self, trade: Trade, order_type: str, reason: str) -> Non
 #
 
     def update_trade_state(self, trade: Trade, action_order: dict = None,
-                           order_amount: float = None) -> None:
+                           order_amount: float = None) -> bool:
         """
         Checks trades with open orders and updates the amount if necessary
         Handles closing both buy and sell orders.
```

Comment:
pls add the description for the return value here -- when it's supposed to return True and False